docs: add CLI section for tctl alert create (#16976)
gravitational/teleport: docs/pages/reference/cli.mdx

    --identity=identity.pem
```

### tctl alerts create

> Creates cluster alerts

Reviewer: ```suggestion
Creates cluster alerts.
```

Reviewer: I think we should add a sentence about what a cluster alert is for users who are skimming the `tctl` reference for all available commands.

> ```$ code

Reviewer: ````suggestion
```code
````

$ tctl alerts create [<flags>] <message>
```

#### Arguments

- `<message>`: The message for the alert to display

#### Flags

| Name | Default Value(s) | Allowed Value(s) | Description |
| - | - | - | - |
| `--severity` | `low` | `low`, `medium`, `high` | Alert severity |
| `--ttl` | none | Any duration | Optional expiry for alert (alert does not expire if no TTL is specified) |

> | `--labels` | none | any | A list of labels to attach to the alert. Labels can be used to tweak alert behavior |

Reviewer: - What are some quick examples of ways that labels can be used to tweak alert behavior?

- How would a user determine what values to supply for `--labels`? Are these the same labels that are already assigned to cluster resources, or something else?

Author: That's covered down below. There are currently two labels (`alert-on-login` and `alert-permit-all`) that affect alert behavior.

    --identity=identity.pem
```

### tctl alerts create

Creates cluster alerts. Cluster alerts can be displayed to Teleport users

> in the web UI or upon logging via `tsh`.

Reviewer: ```suggestion
in the Web UI or upon logging in via `tsh`.
```

```code
$ tctl alerts create [<flags>] <message>
```

#### Arguments

- `<message>`: The message for the alert to display

#### Flags

| Name | Default Value(s) | Allowed Value(s) | Description |
| - | - | - | - |
| `--severity` | `low` | `low`, `medium`, `high` | Alert severity |
| `--ttl` | none | Any duration | Optional expiry for alert (alert does not expire if no TTL is specified) |

> | `--labels` | none | any | A list of labels to attach to the alert. Labels can be used to tweak alert behavior |

Reviewer: ```suggestion
| `--labels` | none | any | A list of labels to attach to the alert, which Teleport uses to configure alert behavior. Teleport recognizes two possible values, which are listed below |
```
When I first reviewed this, I was under the impression that you could supply any string for the `--labels` flag, since the "Allowed Value(s)" column value is "any". 

I've suggested this wording and retaining "any" to communicate that you _can_ supply any value in `--labels`, but Teleport only has logic that supports the two that are listed below.

Author: Got it. I modified your suggestions a bit, but I think we've addressed the confusion.

Yes, any labels can be applied to an alert (they're just arbitrary key/value pairs), however there are some internal labels that can be used to change the way alerts behave.
